main_test.go

```diff
@@ -17,54 +17,84 @@ func TestConfigRejectionCondition(t *testing.T) {
 				App:     "my-app",
 				Version: "0.1.2",
 				Label:   "nightly",
+				Reason:  "no nightlies",
 			},
 			{
 				App: "block-my-app",
 			},
+			{
+				UserTextMatch: "(\\w{4}\\s){11}\\w{4}",
+				Reason:        "it matches a recovery key and recovery keys are private",
+			},
 		},
 	}
 	rejectPayloads := []payload{
```

Comment:
could you add a comment here explaining what `Data.ExpectedRejectReason` is about? It seems a reasonable hack but a clarifying comment might be helpful.